apps/steadyhand/app/dashboard/applications/[id]/prep/page.tsx

```
@@ -0,0 +1,158 @@
+function ChevronRightIcon({ className }: { className?: string }) {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 24 24"
+      fill="none"
+      stroke="currentColor"
+      strokeWidth={2}
+      strokeLinecap="round"
+      strokeLinejoin="round"
+      className={className}
+    >
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)